windows/devices/uwb/UwbCxAdapterDdiLrp.cxx

```diff
@@ -644,6 +644,15 @@ windows::devices::uwb::ddi::lrp::From(const UwbApplicationConfigurationParameter
             applicationConfigurationParameterWrapper = std::make_unique<UwbApplicationConfigurationParameterWrapper>(totalSize);
             UWB_APP_CONFIG_PARAM &applicationConfigurationParameter = applicationConfigurationParameterWrapper->value();
             applicationConfigurationParameter.paramValue[0] = value;
+        } else if constexpr (std::is_same_v<T, std::unordered_set<::uwb::UwbMacAddress>>) {

```

Comment:
Thanks for adding this. Can you please add a test for this case in [`TestUwbCxAdapterDdiLrpConversion.cxx`](https://github.com/microsoft/nearobject-framework/blob/develop/tests/unit/windows/TestUwbCxAdapterDdiLrpConversion.cxx)?